chat/chat.js

```diff
@@ -0,0 +1,32 @@
+const messageContainer = document.querySelector("#message-container")
+
+function addFirstUserMessage() {
+    const messageElement = createMessageElement("first-input");
+    messageElement.className += " main-user";
+    displayMessageInContainer(messageElement)
+    document.getElementById("first-input").value = "";
+}
+
+function addSecondUserMessage() {
+    const messageElement = createMessageElement("second-input");
+    displayMessageInContainer(messageElement)
+    document.getElementById("second-input").value = "";
+}
+
+function createMessageElement(inputId) {
```

Comment:
functions with "create" word usually returns void, for thoselike functions better to use "get" prefix - because this function actually returns some value, that you get from calling it)